src/WikiService.Validation/Rubric/EditRubricRequestValidator.cs

```diff
@@ -0,0 +1,154 @@
+﻿using FluentValidation;
+using FluentValidation.Validators;
+using LT.DigitalOffice.Kernel.Validators;
+using LT.DigitalOffice.WikiService.Data.Interfaces;
+using LT.DigitalOffice.WikiService.Models.Db;
+using LT.DigitalOffice.WikiService.Models.Dto.Requests.Rubric;
+using LT.DigitalOffice.WikiService.Validation.Rubric.Interfaces;
+using Microsoft.AspNetCore.JsonPatch.Operations;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.WikiService.Validation.Rubric
+{
+  public class EditRubricRequestValidator : ExtendedEditRequestValidator<DbRubric, EditRubricRequest>, IEditRubricRequestValidator
+  {
+    private readonly IRubricRepository _rubricRepository;
+
+    private async Task HandleInternalPropertyValidationAsync(
+      Operation<EditRubricRequest> requestedOperation,
+      CustomContext context)
+    {
+      Context = context;
+      RequestedOperation = requestedOperation;
+
+      #region paths
+
+      AddСorrectPaths(
+        new List<string>
+        {
+          nameof(EditRubricRequest.Name),
+          nameof(EditRubricRequest.ParentId),
+          nameof(EditRubricRequest.IsActive)
+        });
+
+      AddСorrectOperations(nameof(EditRubricRequest.Name), new () { OperationType.Replace });
+      AddСorrectOperations(nameof(EditRubricRequest.ParentId), new () { OperationType.Replace });
+      AddСorrectOperations(nameof(EditRubricRequest.IsActive), new () { OperationType.Replace });
+
+      #endregion
+
+      #region Name
+
+      AddFailureForPropertyIf(
+        nameof(EditRubricRequest.Name),
+        x => x == OperationType.Replace,
+        new()
+        {
+          {
+            x => !string.IsNullOrWhiteSpace(x.value?.ToString()),
+            "Name must not be empty."
+          },
+          {
+            x => x.value.ToString().Trim().Length <= 150,
+            "Name is too long."
+          }
+        }, CascadeMode.Stop);
+
+      #endregion
+
+      #region IsActive
+
+      AddFailureForPropertyIf(
+       nameof(EditRubricRequest.IsActive),
+       x => x == OperationType.Replace,
+       new()
+       {
+         {
+           x => bool.TryParse(x.value?.ToString(), out bool _), "Incorrect rubric IsActive format"
+         },
+       }, CascadeMode.Stop);
```

Comment:
I'm not sure.
Do you need "CascadeMode.Stop" here? Here is only one check for IsActive, how would CascadeMode.Stop work?